website/content/docs/enterprise/admin-partitions.mdx

```diff
@@ -96,174 +97,184 @@ This section describes how to deploy Consul admin partitions to Kubernetes clust
 
 The expected use case is to create admin partitions on Kubernetes clusters. This is because many organizations prefer to use cloud-managed Kubernetes offerings to provision separate Kubernetes clusters for individual teams, business units, or environments. This is opposed to deploying a single, large Kubernetes cluster. Organizations encounter problems, however, when they attempt to use a service mesh to enable multi-cluster use cases, such as administration tasks and communication between nodes.
 
-The following procedure will result in an admin partition in each Kubernetes cluster. The Consul clients running in the cluster with servers will be in the `default` partition. Another partition called `clients` will also be created. 
+The following procedure will result in an admin partition in each Kubernetes cluster. The Consul clients running in the cluster with servers will be in the `default` partition. Another partition called `clients` will also be created.
 
 Verify that your Consul deployment meets the [Kubernetes Requirements](#kubernetes-requirements) before proceeding.
 
-1. Verify that your VPC is configured to enable connectivity between the pods running Consul clients and servers. Refer to your virtual cloud provider's documentation for instructions on configuring network connectivity. 
+1. Verify that your VPC is configured to enable connectivity between the pods running Consul clients and servers. Refer to your virtual cloud provider's documentation for instructions on configuring network connectivity.
 1. Create the license secret in each cluster, e.g.:
 
-  ```shell-session
-  kubectl create secret generic license --from-file=key=[license file path i.e. ./license.hclic]
-  ```
-  This step must also be completed for every cluster.
+```shell-session
+kubectl create secret generic license --from-file=key=[license file path i.e. ./license.hclic]
+```
+
+This step must also be completed for every cluster.
 
 1. Create a server configuration values file to override the default Consul Helm chart settings:
 
   <CodeTabs heading="server.yaml">
   <CodeBlockConfig lineNumbers>
 
-  ```yaml
-  global:
-    enableConsulNamespaces: true
-    tls:
-      enabled: true
-    image: hashicorp/consul-enterprise:1.11.1-ent
-    adminPartitions:
-      enabled: true
-    acls:
-      managedSystemACLs: true
-    enterpriseLicense:
-      secretName: license
-      secretKey: key
-  server:
-    exposeGossipAndRPCPorts: true
-  connectInject:
-    enabled: true
-    consulNamespaces:
-      mirroringK8S: true
-  controller:
+```yaml
```

Comment:
I'm not sure this dedent is correct. I see a build failure (https://vercel.com/hashicorp/consul/2c8THwa3ZGRWwaeCDnxpWKSC8BeA), and this might be the cause.

The indentation looks right on the live site: https://www.consul.io/docs/enterprise/admin-partitions

I think we probably want to keep these examples indented so that they appear correctly in the numbered list.  The same applies to the code block above this, and the others below on this page.

Reply:
This page won't render for me locally and also failed to build when I initially pushed the branch. I found that the precommit hook keeps unindenting the lines when saving the commit.

I'm going to spend a little time investigating how to fix that issue. If I can't fix pre-commit hook, I'll just commit without it and that should resolve the build issue.